drydock/plugin.py

```diff
@@ -161,6 +171,7 @@ def get_sync_waves_for_resource(resource_name: str) -> SYNC_WAVES_ORDER_ATTRS_TY
             "superset-celery-beat",
         ],
         "NGINX_STATIC_CACHE_CONFIG": {},
+        "IMAGE_PULL_CONFIG": [],
```

Comment:
```suggestion
        "REGISTRY_CREDENTIALS": "",
```

Just use a string variable that receives the content of `.docker/config.json`

For example 
```yaml
DRYDOCK_REGISTRY_CREDENTIALS: '{"auths":{"https://index.docker.io/v1/":{"auth":"dXNlcjpwYXNz"}}}'